include/seqan3/alignment/pairwise/execution/alignment_executor_two_way.hpp

```diff
@@ -134,49 +134,38 @@ class alignment_executor_two_way
     ~alignment_executor_two_way() = default;
 
     /*!\brief Constructs this executor with the passed range of alignment instances.
+     * \tparam exec_policy_t The type of the execution policy; seqan3::is_execution_policy must return `true`. Defaults
+     *                       to seqan3::sequenced_policy.
      * \param[in] resrc The underlying resource containing the sequence pairs to align.
      * \param[in] fn    The alignment kernel to invoke on the sequences pairs.
+     * \param[in] exec  Optional execution policy to use. Defaults to seqan3::seq.
      *
      * \details
      *
      * Forwards the resource range as a zipped view with an index view to provide internal ids for the alignments.
-     * If the execution handler is parallel it allocates a buffer of the size of the given resource range.
+     * If the execution handler is parallel, it allocates a buffer of the size of the given resource range.
      * Otherwise the buffer size is 1.
      */
+    template <typename exec_policy_t = sequenced_policy>
+    //!\cond
+        requires is_execution_policy_v<exec_policy_t>
```

Comment:
Any reason why this is a type trait and not a concept?

Reply:
Because it makes no sense to define it as a concept since policies don't have a common interface except that they are CRTP base classes

Reply:
I merge this now, because I think the question is answered and there was nothing else, right?